jdk/test/sun/nio/cs/TestIBMBugs.java

```diff
@@ -22,8 +22,10 @@
  */
 
 /* @test
-   @bug 6371437 6371422 6371416 6371619 5058184 6371431 6639450 6569191 6577466
-   @summary Check if the problems reported in above bugs have been fixed
+ * @bug 6371437 6371422 6371416 6371619 5058184 6371431 6639450 6569191 6577466
+ *      8235834
+ * @summary Check if the problems reported in above bugs have been fixed
+ * @modules jdk.charsets
```

Comment:
These two lines are the issue. These are not part of head-stream patch. Please remove them and then this will be a clean patch.